docs/source/tryjupyter.rst

```diff
@@ -0,0 +1,16 @@
+.. _tryjupyter:
+
+===========
+Try Jupyter
+===========
+
+Would you like to try Jupyter in your web browser?
+
+Go to https://try.jupyter.org. No installation is needed.
+
+
+Ready to install Jupyter?
+-------------------------
+If you have tried Jupyter and like it, please use our detailed
+:ref:`installation guide <install>` to install it in your
```

Comment:
I would phrase this as "install it on your computer". I think that "environment" is a bit too vague for new users.